web_frontend/osmaxx/excerptexport/views.py

```diff
@@ -31,10 +32,7 @@ def execute_converters(extraction_order, callback_host, protocol):
     get_authenticated_api_client().create_job(extraction_order, callback_host=callback_host, protocol=protocol)
 
 
-class OrderNewExcerptView(LoginRequiredMixin, FrontendAccessRequiredMixin, FormView):
-    template_name = 'excerptexport/templates/order_new_excerpt.html'
-    form_class = ExcerptForm
-
+class OrderFormViewMixin(FormMixin):
```

Comment:
what do we gain by inheriting? `class OrderFormViewMixin:` would have the same effect, especially since there is no call to super.